vm/src/obj/objproperty.rs

```diff
@@ -265,7 +265,7 @@ impl<'a> PropertyBuilder<'a> {
         }
     }
 
-    pub fn add_getter<I, V, F: IntoPyNativeFunc<I, V>>(self, func: F) -> Self {
+    pub fn add_getter<I, V, VM, F: IntoPyNativeFunc<I, V, VM>>(self, func: F) -> Self {
```

Comment:
Is the rust convention that template parameter names are a single character? In this case maybe we could use `M` instead of `VM`?

Reply:
That's the most common case, but I think rustc's identifier lint rules view generics as type names, e.g. upper camel case.

Reply:
Is it `VM` or `Vm` in this case? probably `Vm`? I thought it is ok because clippy didn't complain about this.